R/build-home-deps.R

```diff
@@ -0,0 +1,38 @@
+data_dependencies <- function(pkg = ".") {
+  pkg <- as_pkgdown(pkg)
+
+  deps <- pkg$desc$get_deps()
+  deps <- deps[order(deps$type, deps$package), c("package", "type")]
+  deps <- deps[deps$package != "R", ]
+
+  recursive <- sort(unique(unlist(tools::package_dependencies(deps$package[deps$type %in% c("Depends", "Imports")]))))
+  recursive <- setdiff(recursive, deps$package)
+  deps <- rbind(deps, data.frame(package = recursive, type = "Recursive"))
+
+  deps$package <- purrr::map_chr(deps$package, package_link)
+  rownames(deps) <- NULL
+
+  deps
+}
+
+package_link <- function(package) {
+  href <- downlit::href_package(package)
+
+  if (is.na(href)) {
+    if (is_base_package(package)) {
+      href <- NA
```

Comment:
For base R packages, we could link to one of the unoffical documentation renderings, e.g.

- `https://r-universe.dev/manuals/PKG.html` (example: https://r-universe.dev/manuals/parallel.html)
- `https://rdrr.io/r/#PKG` (example: https://rdrr.io/r/#parallel)